lib/ansible/module_utils/azure_rm_common.py

```diff
@@ -75,6 +75,8 @@
 CIDR_PATTERN = re.compile(r"(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1"
                           r"[0-9]{2}|2[0-4][0-9]|25[0-5])(/([0-9]|[1-2][0-9]|3[0-2]))")
 
+NEXT_HOP_TYPE = ['VirtualNetworkGateway', 'VnetLocal', 'Internet', 'VirtualAppliance']
```

Comment:
@tomvachon you could move this change to azure_rm_routetable module.
if you don't make changes in common code CI is much faster and simpler, as only your module is tested, and you will also avoid any conflicts with other prs